### ISSUE TITLE
Cannot save table of results

### ISSUE THREAD

**Issue by kzapfe** [neurodroid/stimfit#53]
Hello stimfit team:

I have problems reading the documentation or maybe there is a feature that I cannot understand or is not present.

If I perform an analysis using the graphical interface, and then select the Analysis/Batch Analysis from the Menu, I get this nice table presenting the info that I need. Is there a python object/variable accesible from the Python shell where I can get this list? I guess there must be some sort of dictionary that is created before the window opens with all that info, but I cannot seem to find it in the documentation. My intention is to make this batch analysis automatically on large sets of traces and save the tables as text files or hdf5, according to convenience. 

Thank you for your time.
K.

**Comment by neurodroid** [comment]
@kzapfe Good point, while you can produce a new table using `stf.show_table`, there is currently no way of accessing an existing table.
I wouldn't have time to implement this feature at the moment, but if someone wants to have a go, I'd be happy to provide advice.
